README.md

```diff
@@ -60,19 +48,44 @@ python manage.py db upgrade
 ```
 Rerun this last command after changing the models or checking out a new branch.
 
-For testing, a test database is set up automatically using environment variables. You will need to set the following variables or add them to your `.env` file. Password can be left out if your database is not password protected. See Linux instructions below.
+For testing, a test database is set up automatically. If you want to run tests, you will need to provide the credentials to do this. By default, Alembic will try to set up a 
+test database using the username `postgres`, no password, and no specified port or host. If your setup requires different values, export these environment variables.
+
+
+### Environment management
+
+Project-wide environment variables are set in `.env`. To set them and activate
+the virtual environment by hand, run `source .env`.
+
+Depending on your Postgresql installation, you may also need to export local environment variables.
+By default, Sabelotodo tries to create test databases using the username `postgres`, no password,
+and no specified port or host. To override these defaults, export these environment variables.
 ```
 export DB_HOST=localhost
 export DB_PORT=5432
 export DB_USER=<your postgres username>
-export DB_DATABASE=sabelotodo_test
 export DB_PASSWORD=<your db password>
-export TEST_DATABASE_URL=postgresql:///sabelotodo_test
 ```
+On Linux, do not set the DB_HOST and DB_PORT env variables. Create a superuser
```

Comment:
I like having the `#### Linux` header. It lets other users know that they can skip these instructions and draws the attention of Linux users.